src/Jackett.Common/Definitions/uhdbits.yml

```
@@ -74,7 +77,7 @@ search:
     - path: torrents.php
   inputs:
     $raw: "{{ range .Categories }}filter_cat[{{.}}]=1&{{end}}"
-    searchstr: "{{ .Keywords }}"
+    searchstr: "{{ .Keywords }}{{ if .Query.Season }} Season {{ .Query.Season }}{{ end }}"
```

```suggestion
    searchstr: "{{ .Keywords }}"
```

This would result in `Show Name S01E02 Season 1`. You could use `.Query.Q` instead of `.Keywords`, but this is used nowhere else and we're already using `keywordsfilters` below to convert `S01E02` or `S01` to `season 1`.

You could edit the `keywordsfilters` comment below to:

```yml
    # S01E02 or S01 > season 1 #10423
```

---

> This would result in Show Name S01E02 Season 1.

Only if the searching client set both the `imdbid`, `season` and the `q` attributes simultaneously, right? I don't believe Sonarr does this. Even so, `Season 1 Season 1` shouldn't cause any problems.

---

Sorry for the delay, was putting food in my face there.

Using the above (with the missing `{{ else }}` added):
`http://127.0.0.1:9117/api/v2.0/indexers/bitsearch/results/torznab/api?apikey=xxxxx&t=search&cat=&q=Show+Name&season=1&ep=2`
results in:
`searchstr=Show+Name+S01E02+Season+1`
or with the keywordsfilters:
`searchstr=Show+Name+season+1+Season+1`